core/util/src/main/java/edu/cmu/cs/dennisc/javax/swing/event/PausableDocumentListener.java

```diff
@@ -0,0 +1,32 @@
+package edu.cmu.cs.dennisc.javax.swing.event;
+
+import edu.cmu.cs.dennisc.java.util.logging.Logger;
+
+import javax.swing.event.DocumentEvent;
+import java.util.function.Consumer;
+
```

Comment:
this new class, i love it. 

maybe a comment about when you want to use this vs the unpausable, but i'm being nitpicky.

Reply:
I made it because the pattern is used.
Just when you would **want** to use it I am less sure of. Determining that might require deeper analysis, or asking the original developers.